includes/class-amp-story-legacy-post-type.php

```diff
@@ -44,9 +44,10 @@ class AMP_Story_Legacy_Post_Type {
 	/**
 	 * The rewrite slug for this post type.
 	 *
+	 * @todo Figure out a suitable rewrite slug for this.
 	 * @var string
 	 */
-	const REWRITE_SLUG = 'stories';
+	const REWRITE_SLUG = 'amp-stories';
```

Comment:
Changed this since both the new and the old Story CPT-s were using the same at this moment.

Reply:
Good spot :+1: